.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "samples"]
+	path = samples
+	url = https://gitlab.office.flavoursys.com/internal/view-samples.git
```

Comment:
We should probably open source that repository 😄

Reply:
Yes, I thought the same :smile: Totally forgot that I used the samples from there. 
Do you know which license governors the samples?

Reply:
In the end I moved it to github :)